base/gocb_connection_string.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2024-Present Couchbase, Inc.
+//
+// Use of this software is governed by the Business Source License included
+// in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+// in that file, in accordance with the Business Source License, use of this
+// software will be governed by the Apache License, Version 2.0, included in
+// the file licenses/APL2.txt.
+
+package base
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/couchbaselabs/gocbconnstr"
+)
+
+const (
+	kvPoolSizeKey = "kv_pool_size"
```

Comment:
I like the idea of making this a constant, I think it makes sense to do the same for kv_buffer_size and dcp_buffer_size?